README.md

```diff
@@ -18,35 +18,50 @@ these checks verify that the plugin is available for users of napari
 to install, and would register entry points with napari. 
 
 There are two parts to the validation tool. One part is a 
-command line interface and the other is a pytest fixture.
+command line interface, and the other is a pytest fixture.
 
 ### Command Line Interface (CLI) Usage
 The CLI can be used by continuous integration (CI) pipelines to perform a 
-quick verification of a plugins setup without any specific input required. 
-It serves as a quick "sanity check". (It is also accessible from python in validation.py)
+quick verification of a plugin setup without any specific input required. 
+It serves as a quick "sanity check". (It is also accessible via Python in validation.py)
 
 The tool can be run as `npd <cmd>`. where currently we support cmd `validate`:
 
-`npd validate`: validate classifiers and function hooks can be recognized by napari.
-The validation would run on built packages under dist folder to check if they are annotated
-properly with framework classifier, and validate hooks are properly annotated that they can be found
-by napari.
+Before running `npd validate`, make sure you have prepared the repo for validation:
+1. install the plugin to the same python environment where `npd validate` is run:
+   ```
+   cd <plugin path>
+   pip install napari-plugin-devtools
+   pip install -e .
+   ```
+2. package the plugin, if skipped we do not run classifier validation, which may cause your plugin not showing up in 
+   napari native plugin installation menu. The build process can be different for your plugin, for a typical setup try:
+   ```
+   python setup.py sdist bdist_wheel
+   ```
 
-`npd validate -i|--include-plugin INCLUDE_PLUGIN [INCLUDE_PLUGIN ...]` run hook checks only on listed plugins, 
-this is useful to filter out other plugins on a complicated python environment.
+`npd validate` list function hooks under current python environment, to install a plugin from a repo to python 
+environment, run `pip install -e <plugin path>`
 
+`npd validate -d|--dist dist` name of the dist, default to first plugin name if not provided, should be provided when 
+there are multiple plugins in same package or plugin name is different from package name
 
-`npd validate -e|--exclude-plugin EXCLUDE_PLUGIN [EXCLUDE_PLUGIN ...]` do not run hook checks on listed plugins, 
-this is useful to filter out other plugins on a complicated python environment.
+`npd validate -v|--verbose` enable the verbose mode, gives slightly more information on the underlying 
+findings of validation process. It is by default disabled to avoid the eyesore from too much text.
 
-`npd validate -v|--verbose` enable verbose mode, gives slightly more information on the underlying 
-findings of validation process.
+Output from `npd validate` has multiple sections, where each section is a separate validation where the header
```

Comment:
I like that you explain what each section is for so I know what will be validated. I think the order here is actually different than the output though which could be confusing. It would be good to have these checks in the same order as the output when I run  the validate command.